ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -1,40 +1,39 @@
+import { Injectable, Inject, OnDestroy } from '@angular/core';
 import { ConfigurationService } from './configuration.service';
 import { LogEvent } from '../models/logEvent';
-import { Injectable, Inject, OnDestroy, OnInit } from '@angular/core';
 import { LogLevel } from '../models/logLevel';
 import { DdpException } from '../models/exceptions/ddpException';
 import { ExceptionDispatcher } from './exceptionHandling/exceptionDispatcher.service';
 import { Observable, Subscription, ReplaySubject } from 'rxjs';
 import { filter } from 'rxjs/operators';
 
 @Injectable()
-export class LoggingService implements OnInit, OnDestroy {
+export class LoggingService implements OnDestroy {
     private subjectSource = new ReplaySubject<LogEvent>();
     private anchor: Subscription;
 
     constructor(
-        @Inject('ddp.config') private config: ConfigurationService,
-        private dispatcher: ExceptionDispatcher) {
-    }
-
-    public ngOnInit(): void {
```

Comment:
The problem was that Angular services don't have ngOnInit lifecycle hook. Their initialization should be in the constructor.